doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md

```diff
@@ -65,7 +65,7 @@ Configure the Docker services accordingly:
 
```

Comment:
Should we change the Chromium (Chrome Headless) paragraph to look more optional and interchangeable with Gotenberg, maybe under a common chapter with 2 potential approaches? 
https://github.com/pimcore/pimcore/blob/a94528de01619269c1ce6c00f69d99b1434b8c69/doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md?plain=1#L34-L41

Reply:
Done - thanks! 
I've also added some deprecation info.